scripts/tests/deployment_test.js

```diff
@@ -0,0 +1,225 @@
+/**
```

Comment:
I assume you just moved the file. Next time (or, if you feel like it, you can fix it with a quick rebase) use `git mv` instead, I think github will show us the diff automatically.

Reply:
yes. I just moved it. next time will do it with git mv.